src/app/notifications/page.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import { User } from "lucide-react";
+import Navbar from "../_components/Navbar";
+
+interface Notification {
+	id: number;
+	message: string;
+	action: string;
+}
+
+const mockNotifications: Notification[] = [
+	{
+		id: 1,
+		message: "Muhammad Saleh liked your project idea",
+		action: "View Post",
+	},
+	{
+		id: 2,
+		message: "Mykhalio Chudyk left a comment on your project idea",
+		action: "View Post",
+	},
+	{
+		id: 3,
+		message: "Nour Elafia liked your project idea",
+		action: "View Post",
+	},
+];
+
+const Notifications: React.FC = () => {
+	const [notifications, setNotifications] = useState<Notification[]>([]);
+
+	useEffect(() => {
+		setNotifications(mockNotifications);
+	}, []);
+
+	return (
+		<div className="flex min-h-screen flex-col items-center justify-center bg-gradient-to-tr from-[#F7F7F8] via-[#E3E7FF] to-[#DCE0FF] p-4">
+			<Navbar />
+			<div className="w-full max-w-lg rounded-xl border border-indigo-200 bg-white p-6 shadow-lg">
+				{notifications.map((notification, index) => (
```

Comment:
I could see how the notifications could be broken down into individual components, but we can ignore this for now. It looks good this way too!